websites/serializers.py

```diff
@@ -222,6 +222,12 @@ def update(self, instance, validated_data):
             update_youtube_thumbnail(
                 instance.website.uuid, validated_data.get("metadata"), overwrite=True
             )
+        if "file" in validated_data:
+            if "metadata" not in validated_data:
+                validated_data["metadata"] = {}
+            validated_data["metadata"]["file_type"] = detect_mime_type(
```

Comment:
Just realized this will insert `file_type` into the metadata even if that field is not part of the starter config.  Doesn't seem like this will necessarily break anything though.  Do you think its worth checking first to see if this field is in the config definition, and/or if the website is an ocw course (starter.slug == settings.OCW_IMPORT_STARTER_SLUG)?

Ditto for the `WebsiteContentCreateSerializer`

Reply:
I don't know. Any thoughts @Ferdi @gumaerc @pdpinch?

Reply:
mime type seems like a generally useful thing, when uploading a file. I don't think we need to make it conditional on presence in the starter.